### PR TITLE
dev-libs/zziplib: 0.13.71 version bump

### PR DESCRIPTION
**Pull request by vaukai**
Closes: https://bugs.gentoo.org/723418
Thanks to Nico R. for the ebuild attached to the bug
Adding myself as a proxied maintainer

Package-Manager: Portage-2.3.103, Repoman-2.3.23
Signed-off-by: Volkmar W. Pogatzki <gentoo@pogatzki.net>

### CODE REVIEW

**Comment by vaukai** [comment]
Hm, I am not getting that error:  https://gist.github.com/vaukai/759c247d57418dbd17b7774fdee9a360
emerge --info '=dev-libs/zziplib-0.13.71' https://gist.github.com/b5b0cf62849bb0dd204063ae6353fcc8

**Comment by juippis** [comment]
Try with `USE="doc"` it seems to trigger it.

**Comment by vaukai** [comment]
> Try with `USE="doc"` it seems to trigger it.

Indeed, I should have tested more carefully:

 * Working in BUILD_DIR: "/var/tmp/portage/dev-libs/zziplib-0.13.71/work/zziplib-0.13.71_build"
ninja -v -j8 -l0
ninja: error: build.ninja:1735: bad $-escape (literal $ must be written as $$)

No clue how to fix it :-(

**Comment by juippis** [comment]
Always check upstream, there are most likely other people who have ran to the same issue. 
https://github.com/gdraheim/zziplib/issues/88 this seems to have a patch available for you.

**Comment by vaukai** [comment]
> Always check upstream, there are most likely other people who have ran to the same issue.
> [gdraheim/zziplib#88](https://github.com/gdraheim/zziplib/issues/88) this seems to have a patch available for you.

Thanks for pointing me there.  Need some time to read that carefully

File: dev-libs/zziplib/zziplib-0.13.71.ebuild [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	sdl? ( >=media-libs/libsdl-1.2.6 )
+"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/zziplib-0.13.69-009-perror.patch
+	"${FILESDIR}"/zziplib-0.13.71-497c159.patch
+	"${FILESDIR}"/zziplib-0.13.71-find-bash.patch
+	"${FILESDIR}"/zziplib-0.13.71-ddaa538.patch
+	"${FILESDIR}"/zziplib-0.13.71-75e22f3.patch
```
Comment (juippis) [inline dev-libs/zziplib/zziplib-0.13.71.ebuild:40]: Sorry again but we really want the patch filenames to identify what they're doing. `perror` and `find-bash` are good enough, perror could be better but sufficient. 

https://dev.gentoo.org/~vapier/clean-patches

Comment (vaukai) [reply dev-libs/zziplib/zziplib-0.13.71.ebuild]: > perror could be better but sufficient.
> 
Filename "zziplib-0.13.69-009-perror.patch"  is used already in older ebuild.
497c159, ddaa538 and 75e22f3 renamed.

**Pull request closed without merging**